platform/windows/FindOSMesa.cmake

```diff
@@ -0,0 +1,65 @@
+# Standard FIND_PACKAGE module for OSMesa, sets the following variables:
+#   - OSMesa_FOUND
+#   - OSMesa_INCLUDE_DIRS (only if OSMesa_FOUND)
+#   - OSMesa_LIBRARIES (only if OSMesa_FOUND)
+
+# Try to find the header
+find_path(OSMesa_INCLUDE_DIR NAMES GL/osmesa.h)
+
+get_filename_component(OSMesa_DIR ${OSMesa_INCLUDE_DIR}/../ ABSOLUTE)
+
+# Try to find the library
+if(CMAKE_SIZEOF_VOID_P EQUAL 8)
+    set(_ARCH x64)
+else()
+    set(_ARCH x86)
```

Comment:
will the eventually need a arm64 if we want to support that?

Reply:
I don't see why not, but I really don't have tools for developing that. But theorically yes, it's possible.